packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -93,6 +93,10 @@ spec:
                   fieldPath: metadata.namespace
             - name: STRIMZI_FEATURE_GATES
               value: ""
+            - name: STRIMZI_OPERATOR_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
@scholzj - IIRC I also have to add this to the Helm charts, is that right?

Reply:
You need to make the change to the Helm Chart first. And then when you run `make helm_install` it should propagate the change into the YAML installation files.

Reply:
Cheers @scholzj!